doc/nrf/templates/Ecosystem_integration_template_README.rst

```diff
@@ -0,0 +1,130 @@
+.. _Ecosystem_integration:
+
+Template: Ecosystem integration
+###############################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. note::
+   Provide a short introductory description of the tool or service (third-party), indicating the value it brings to |NCS|.
+   Refer to the following instructions to create the description:
+
+   * Provide a title that starts with the tool or service name with "integration" as a suffix, for example, "Memfault integration".
+   * Place the documentation inside the :file:`nrf/doc/nrf` folder.
+   * List the file name in the ``.. toctree:`` of the :file:`ext_comps` RST file.
+   * Sections with * are optional and can be left out.
+     All other sections are required.
+     Do not add new sections (unless in the sections that allow for further subsections) without discussion with the tech writer team.
+
+.. tip::
+   Describe the tool or service in one or two sentences maximum (full sentences, not sentence fragments), so that the user gets a clear idea of what the tool or service does.
+   This introduction must be concise and clear, highlighting the features of the tool or service that it brings to |NCS|.
+
+Partner overview*
+*****************
+
+.. note::
+   Provide an explanation of the tool or service.
+   This is optional if the tool or service is described above.
+
+Prerequisites
+*************
+
+.. note::
+   * Provide prerequisites that must be completed before the integration with the tool or service.
+   * Mention any other tool or service integration that needs to be completed.
+
+Before you start the |NCS| integration with AVSystem's Coiote IoT Device Management, make sure that the following prerequisites are completed:
```

Comment:
I would prefer to keep it as is because I think there can be scenarios where the prerequisites are already set up (for example, an existing user would have completed the NCS installation).